batchflow/models/torch/layers/pixel_shuffle.py

```diff
@@ -0,0 +1,83 @@
+""" Layer implementing Pixel Shuffle for an arbitrary dimensionality.
+Shi et al. "`Real-Time Single Image and Video Super-Resolution
+Using an Efficient Sub-Pixel Convolutional Neural Network_
+<https://arxiv.org/pdf/1609.05158.pdf>`"
+"""
+from torch import nn
+
+
+
+class PixelShuffle(nn.Module):
+    """ Rearranges elements in a tensor of shape [B, C * r^N, D1, D2, ... DN]
```

Comment:
I do like the idea of copy-paste the documentation from `PyTorch` counterpart, but maybe something along the lines of `just as in PyTorch, but works with arbitrary number of dimensions` would be helpful